helm/coredns-app/templates/deployment-masters.yaml

```diff
@@ -12,6 +12,7 @@ spec:
   selector:
     matchLabels:
       {{- include "labels.selector" . | nindent 6 }}
+      app.kubernetes.io/component: control-plane
```

Comment:
not sure "component" is the right label here.
It is more about "destination node".
Is there any upstream label which has a better meaning?

Reply:
Nah that was the closest.

Reply:
How about `target: workers` ?

Reply:
yeah I prefer target or `target-node` maybe.

Reply:
`target-node` sounds like it's a specific node, not a "type" of node.

Reply:
agreed